.mega-linter.yml

```diff
@@ -1,16 +1,12 @@
 APPLY_FIXES: none
-FILTER_REGEX_EXCLUDE: (karma.config.js|polyfill.js|timeout.js|CHANGELOG.md)
```

Comment:
Please add to this line the test/ not just the test/

Reply:
that's not required, it won't validate js stuff, the `test` exclude is for the e2e test